app/routes/routes.py

```diff
@@ -313,17 +368,64 @@ def valid_username(username):
 
 
 def valid_question(questionObject):
-    if 'topic' in questionObject and 'body' in questionObject:
+    if 'topic' in questionObject.keys() and 'body' in questionObject.keys():
```

Comment:
You can shift these validator functions to a different file and import them to use them in the views so that the routes.py has routes only